### ISSUE TITLE
Picture synchronization is broken with SoGo and Cardbook (Thunderbird)

### ISSUE THREAD

**Issue by clochix** [cozy/cozy-sync#135]
Reported by two users on the forum ([here](https://forum.cozy.io/t/impossible-de-synchroniser-les-contacts/2058/34) and [here](https://forum.cozy.io/t/synchro-des-contacts-avec-carddav-et-thunderbird-problemes-dimages-de-contacts-entre-autres/2984)). I can confirm issues trying to sync some contact between Cozy and Thunderbird, using either SoGo or Cardbook.

**Comment by jsilvestre** [comment]
I did more testing on this: 
- I could create a contact on both ends with a picture and it was properly synchronized
- It doesn't work when I **edit** a picture in CardBook and then I sync it: all the fields are synced except the picture.

**Comment by fzinken** [comment]
@jsilvestre I don't think this is a CardBook specific topic. Maybe the other user had the issue with Thunderbird. I was just testing between my Android Tablet and Cozy Webinterface and had that issue.
Basically created two contacts on the tablet (one with and one without picture) and also two (one with and one without picture) on the server. The result is I have 4 contacts on the tablet but only 3 on the server.
I had uploaded application logs at that time.

**Comment by Flore-mozfr** [comment]
As I described in the post on cozy forum, I have the issue everywhere:

> Sur la tablette (Android 5.1) ou le téléphone (Android 6), j'ajoute une image (photo ou identicon, c'est pareil), je synchronise avec DavDroid et ça supprime l'image (ou bien ça remet l'ancienne), par contre ça va prendre en compte les autres modifications (anniversaire, ajout d'adresse ou de numéro de téléphone). Je peux tout à fait créer et modifier des contacts sans problème, tant que je ne mets pas d'image. C'est pénible, quand même.

So I suspect the problem comes from how cozy handles carddav sync. Before cozy, I used the same config with a carddav server on synology and had no issue with pictures for contacts on android, sogo, thunderbird or whatever.
